eng/pipelines/templates/pipeline.yml

```diff
@@ -106,39 +107,39 @@ stages:
         swix:
           enabled: true
         optprof:
-          enabled: "and(succeeded(), eq(variables['BuildRTM'], 'false'), eq(variables['IsOfficialBuild'], 'true'))"
```

Comment:
Why is the BuildRTM unnecessary? 
Wouldn't it now run  in 2 phases while it previous ran in 1?

Reply:
line: 95 hardcodes it.

Reply:
Line 95 sets up a job variable, but it doesn't prevent this optprof step to skip on official build's RTM job.

Reply:
> @zivkan could you rephrase this, please? Optprof only runs on NonRTM

Github is showing me the following diff on the files tab, but for some reason not on the conversation tab:

```diff
-enabled: "and(succeeded(), eq(variables['BuildRTM'], 'false'), eq(variables['IsOfficialBuild'], 'true'))"
+enabled: ${{ variables['isOfficialBuild'] }}
```

So previously it would only run on the non-RTM job, and only on official builds.

Now, it looks like it's running on official builds for any job.

However, while gathering evidence, I now realize this is part of the nonRTM job defined on line 87, whereas the RTM job definition starts on 271.  When I originally split up the jobs into different files, I intended the individual yamls to contain "all" the steps, including publishing, so now that pipeline.yml has all the outputs, it's completely thrown off my mental model for how the CI jobs are defined.